test/100-bridge-iptables.bats

```diff
@@ -148,6 +148,37 @@ fw_driver=iptables
     run_netavark --file ${TESTSDIR}/testfiles/ipv6-bridge.json teardown $(get_container_netns_path)
 }
 
+@test "$fw_driver - bridge driver must generate config for aardvark with custom dns server" {
+    # get a random port directly to avoid low ports e.g. 53 would not create iptables
```

Comment:
just skip the test for now, then we can merge. get aardvark-dns tests working and once aardvark-dns is merged we can enable this test.

Reply:
It looks to me that `aardvark`'s integration test is using netavark

Reply:
ah i see what you mean, let me skip this for now.

Reply:
**Note!!!**: following test must be unskipped in another PR after https://github.com/containers/aardvark-dns/pull/240